### PR TITLE
CLDR-15625 always allow TC to vote for 4/6

### PR DESCRIPTION
**Pull request by srl295**
CLDR-15625

- [x] This PR completes the ticket.

### CODE REVIEW

**Comment by srl295** [comment]
image

(i've added 1-vote also since then.  TC menu.)

**Review by btangmu — APPROVED** [review]

File: tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java [hunk 1]
```diff
@@ -287,8 +297,11 @@ public ImmutableSet<Integer> getVoteCountMenu(Organization org) {
          * vetter.votes needs to be defined before we can set admin.voteCountMenu.
          */
         static {
-            admin.voteCountMenu = ImmutableSet.of(vetter.votes, admin.votes); /* Not LOCKING_VOTES; see canVoteWithCount */
-            tc.voteCountMenu = ImmutableSet.of(vetter.votes, tc.votes, PERMANENT_VOTES);
+            admin.voteCountMenu = ImmutableSet.of(
+                street.votes, vetter.votes, vetter.tcorgvotes, tc.votes, admin.votes, PERMANENT_VOTES);
+            /* Not LOCKING_VOTES; see canVoteWithCount */
+            tc.voteCountMenu = ImmutableSet.of(
+                street.votes, vetter.votes, vetter.tcorgvotes, tc.votes, PERMANENT_VOTES);
```
Comment (srl295) [inline tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java:304]: @macchiati  @stenshamn  can you confirm this set of votes for TC?
- 1 (street)
- 4 (vetter) 
- 6 (TC org vetter)
- 50 (TC)
- 1000 (PERMANENT_VOTES)

Comment (stenshamn) [reply tools/cldr-code/src/main/java/org/unicode/cldr/util/VoteResolver.java:304]: Looks right to me.

**Comment by macchiati** [comment]
Yes

**Pull request merged**